my-app/src/components/login/login.js

```diff
@@ -41,6 +41,28 @@ class Login extends React.Component {
     .catch(err => console.error(err));
   }
 
+  getUserServer = _ => {
```

Comment:
This file has various eslint errors, make sure to run eslint on every file you update/work on

Reply:
I'll look into it

Reply:
Fixed this issue in latest commit.